src/background.js

```diff
@@ -524,6 +524,7 @@ function handleGhosteryHub(name, message, callback) {
 			const origin = message.origin || '';
 			if (origin === 'onboarding') {
 				conf.setup_step = message.setup_step;
+				conf.setup_number = message.setup_number;
```

Comment:
Even though we use the same Metrics qs param for both, can we store this `setup_number` in a new and separate conf prop to make the whole thing a bit less dirty? Maybe `dawn_setup_number`. Then in `Metrics` we would `buildQueryPair` with one or the other depending on which ping we were sending

Reply:
Good idea, changed to `dawn_setup_number` and made a function that cases on `gb_onboarding`